dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -115,6 +115,18 @@
       )
     %p= t('user.only_one_parent_email_supported_warning')
 
+- if experiment_value('ai_rubrics', request)
```

Comment:
ha, I named this experiment `ai-rubrics`. We should probably standardize. Happy to use `ai_rubrics` as I'll need to update my PR anyway